quic/s2n-quic-transport/src/path/manager.rs

```diff
@@ -90,7 +90,7 @@ impl<Config: endpoint::Config> Manager<Config> {
         new_path_id: Id,
         random_generator: &mut dyn random::Generator,
         publisher: &mut Pub,
-    ) -> Result<(), transport::Error> {
+    ) -> Result<bool, transport::Error> {
```

Comment:
would it be worth having an `enum` for this instead? It's not super clear from the function signature what callers are supposed to do with this value.

I would also put a `#[must_use]` on the enum type so we don't miss it

Reply:
yep, good call